client/components/RelatedItems.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import $ from 'jquery';
+
+class RelatedItems extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.renderScrollButtons = this.renderScrollButtons.bind(this);
+    this.removeScrollButtons = this.removeScrollButtons.bind(this);
+  }
+
+  renderScrollButtons() {
+    $('.scrollButton')
+      .css('border-color', '#666666')
+      .css('color', 'black');
+  }
+
+  removeScrollButtons() {
+    $('.scrollButton')
+      .css('border-color', 'transparent')
+      .css('color', 'transparent');
+  }
+
+  render() {
+    return (
+      <div id="recommendationsProducts">
+        <div
+          class="scrollDiv1"
+          onMouseOver={this.renderScrollButtons}
+          onMouseLeave={this.removeScrollButtons}
+        >
+          <button class="scrollButton">&#8249;</button>
+        </div>
+        <ul id="relatedItemsList">
+          {this.props.relatedProducts.map(item => {
+            let AppleProduct = () => {
```

Comment:
should this be `AppleProduct`? that sounds like an awfully specific variable name that's pegged to one brand